odoo/addons/test_new_api/tests/test_unity_read.py

```diff
@@ -763,6 +763,28 @@ def test_reference_without_values(self):
             }
         ])
 
+    def test_reference_id_without_model(self):
+        self.course.m2o_reference_model = False
+        read = self.course.web_read(
+            {
+                'm2o_reference_id':
+                    {
+                        'fields':
+                            {
+                                'display_name': {},
+                                'write_date': {},
+                            },
+                    },
+                'm2o_reference_model': {}
+            })
+        self.assertEqual(read, [
+            {
+                'id': self.course.id,
+                'm2o_reference_id': self.course.m2o_reference_id,
```

Comment:
The result doesn't make lot of sense to me. For nonexistent record we return False for `m2o_reference_id` and for `m2o_reference_model`. IMO, you should have the same behavior here: no `m2o_reference_model` => `m2o_reference_id` should be `False` too.

Reply:
I thought that this would mean more changes, and make it more difficult to track/fix problems, but I understand the point about consistency. I'm OK with both so I'll comply :)